2022-08-august/README.md

```diff
@@ -77,23 +77,34 @@ Feel free to reach out to anyone here on Slack, and post updates about your own
 
 Yes, I'm talking about Assemble. Or rather, Ian (14) and Sam (17) are:
 
-### TODO: Insert some catchy title for the hackathons grant>
+### 💸 Grants!!! For your hackathon!!! 💸
+![You get $500, Oprah meme](https://cloud-ncs2ypbyu-hack-club-bot.vercel.app/06qcy84.jpeg)
 
 
-### Fun things and trends
+At Assemble, <img src="https://ca.slack-edge.com/T0266FRGM-U0266FRGP-7f1bd1a815ac-512" width="16"> [Zach](https://zachlatta.com/) announced that [Hack Club and FIRST are offering a **✨ $500 grant ✨**](https://hackclub.com/hackathons/) to in-person high-school hackathons hosted before the end of 2022!!! All you need for the grant is a high-school team, a venue, and a custom website, and you get free tools and support too. Learn more at [hackclub.com/hackathons](https://hackclub.com/hackathons) and [APPLY HERE!](https://airtable.com/shrpYdJpAOphEWXan)
 
-TODO
 
-\
+
+### Things to do while procrastinating on math homework:
+- Add your own Orpheus drawing (there are HUNDREDS!!): [github.com/hackclub/dinosaurs](https://github.com/hackclub/dinosaurs)
+- Grab some popcorn 🍿 to join [#movie-nights](https://hackclub.slack.com/archives/C025QEVNRPW), find a quick laugh at [#memes](https://hackclub.slack.com/archives/CB19EP4MC), explore the Hack Club Slack [#hall-of-fame](https://hackclub.slack.com/archives/C028VGT0JMQ), or meet other Hack Clubbers on [#hack-night](https://hackclub.com/night/)
+- Code something small and special in html, css, and/or js (no frameworks) and share it with the community through [freewrite](https://hackclub.slack.com/archives/C03TE1RLR7V/p1660684775476709)
+- Play games 🎮 made by fellow Hack Clubbers (or code your own) using Sprig, a beginner-friendly game engine: [Sprig Gallery](https://sprig-gallery.hackclub.dev/)
+- Join [#amas](https://hackclub.slack.com/archives/C03QPB0SU3V) and help <img src="https://ca.slack-edge.com/T0266FRGM-U03Q20XM953-91ae3b0d0243-512" width="16"> [Mark](https://www.youtube.com/watch?v=dQw4w9WgXcQ) bring amazing guests to [Hack Club's AMAs](https://hackclub.com/amas) 😎
+- Do some good-old lurking 🕵️ and find out if there are other Hack Clubbers near you 📍! Hack Clubbers have been meeting up ([#bayarea](https://hackclub.slack.com/archives/C014M6C0CNA), [#seattle](https://hackclub.slack.com/archives/C023L02KGHZ), [and more](https://hackclub.slack.com/archives/C03L35R822Y/p1660101850115939)) and it looks like so much fun.
+
+![Cool orpheus](https://cloud-9pj8ga86m.vercel.app/2020-07-24_zdd8ycnkp9q0bbf1fj8a1amjv3zndufz.png)
```

Comment:
This was to be used in the hack night reference in the same format as those 16px PFPs

Reply:
ok :)